include/oneapi/dpl/internal/scan_by_segment_impl.h

```diff
@@ -0,0 +1,372 @@
+/*
+ *  Copyright (c) Intel Corporation
+ *
+ *  Copyright 2008-2013 NVIDIA Corporation
+ *
```

Comment:
Do any other reviewers have any insight on what should be changed in the comment header since Mikhail is OOO? It seems that we have two formats used in the codebase: this format which is seen in current mainline scan_by_segment, reduce_by_segment, and several other headers in this internal directory, and the format seen in the pstl directory and beyond.

In addition to the format itself being different, the format I am using lists the license as Apache 2.0 and the other header format uses Apache 2.0 with LLVM exception. Which is the correct to use here? @akukanov @rarutyun @SergeyKopienko

Reply:
I have discussed with @ValentinaKats and have updated the header comment accordingly to provide the correct attributions.